hephaistus/main-theme/distribution-system/chord/playground/zozond/chord-impl/build.gradle.kts

```diff
@@ -23,6 +23,6 @@ kotlin {
     jvmToolchain(17)
 }
 
-application {
-    mainClass.set("MainKt")
-}
\ No newline at end of file
+// application {
+//    mainClass.set("MainKt")
+// }
```

Comment:
Please remove lines if it is not used now. Please do not use temporary comment lines.